content/altinn-studio/guides/development/prefill/_index.en.md

```diff
@@ -6,7 +6,7 @@ weight: 30
 aliases:
 - /altinn-studio/guides/prefill/
 ---
-As of today, Altinn offers three methods for prefilling data in an app for a user.
+As of today, Altinn offers four methods for prefilling data in an app for a user.
```

Comment:
I would probably just change this to 'multiple'. :joy: Feels like such a chore having to maintain stuff like this! :shrug: